src/main/java/org/opentripplanner/transit/model/site/Station.java

```diff
@@ -140,6 +142,13 @@ public ZoneId getTimezone() {
     return timezone;
   }
 
+  /**
+   * If true do not allow any transfers to or from any stop within station
+   */
+  public boolean isNoTransfers() {
```

Comment:
Is `transfersNotAllowed()` a better name ? 

`noTransfers` is not a property on Station, it is a condition/business rule.

Reply:
I changed the name